instrumentation/vertx/vertx-sql-client-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/vertx/v4_0/sql/VertxSqlClientRequest.java

```diff
@@ -8,16 +8,16 @@
 import io.vertx.sqlclient.SqlConnectOptions;
 
 public final class VertxSqlClientRequest {
-  private final String statement;
+  private final String rawQueryText;
```

Comment:
(and elsewhere in this file)
```suggestion
  private final String queryText;
```